app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt

```diff
@@ -268,6 +276,41 @@ open class FileListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
         getBackNavigationResult<Int>(REFRESH_FAVORITE_FILE) {
             mainViewModel.refreshActivities.value = true
         }
+
+        getBackNavigationResult<Int>(DownloadProgressDialog.OPEN_BOOKMARK) { fileId ->
+            FileController.getFileProxyById(fileId, customRealm = mainViewModel.realm)?.openBookmarkIntent()
+        }
+    }
+
+    private fun File.openBookmarkIntent() {
+        kotlin.runCatching {
```

Comment:
Why adding the `kotlin.` prefix?

Reply:
It's not important, it's the same thing, just that it's the auto import that does that, having to change it manually all the time